css-canvas-medical/index.html

```diff
@@ -0,0 +1,109 @@
+              <button class="contact-us">Contact Us</button>
+            </nav>
+          </div>
+        </header>
+      </div>
+      <div class="row justify-center">
+        <div class="hero-img-bg">
+          <div class="overlay">
+            <h1 class="paragraph-title">The EMR Reimaged for <br>Primary Care</h1>
+                  <p class="paragraph line-height">Can't remember the last time you talked with a patient without
+              focusing<br> on the EMR? Canvas lets you rediscover joy in the work
+              you do. Capture<br> meaningful data, identify gaps in care, and move
+              seamlessly between<br> tasks within the platform. You stay present with
+              patients while your<br> whole care team stays up-to-speed. Be a better
+              doctor with Canvas,<br> designed for primary care providers with the
+              help of primary care<br> providers.
+            </p>
+            <button class="get-started top-button">Get Started</button>
+          </div>
+        </div>
+      </div>
+
+
+
+      <div class="row justify-center off-white-bg">
+        <div class="column-full padding-separator">
+          <h1 class="gray-text">Clinicians Prefer Canvas</h1>
+          <div class="line-height text-align-center gray-text">Physician-owned primary care practices across the country have
+          <br>already made Canvas the fabric of their care</div>
+        </div>
+      </div>
+      <div class="row justify-center gray-text off-white-bg bottom-border">
+        <div class="column-one-third text-align-center line-height-three margin-separator">
+          <div>Cascade Family Practice, Portland OR</div>
+          <div>Temescal Creek Medicine, Oakland CA</div>
+          <div>San Juan Healthcare, Friday Harbor WA</div>
+        </div>
+        <div class="column-one-fourth"></div>
+        <div class="column-one-third text-align-center line-height-three margin-separator">
+          <div>Ward Medical Clinic, Bonifay FL</div>
+          <div>Caribou Medical, Towanda PA</div>
+          <div>Bay Tree Family Care, Red Bay AL</div>
+        </div>
+      </div>
+      <div class="row justify-between align-center gray-text page-footer">
+        <div><h1>Ready to learn more?</h1>Schedule a private meeting now</div>
+        <button class="get-started">Get Started</button>
+      </div>
+      <div class="row dark-bg white-text padding-separator-big line-height">
+        <div>
+            <div class="column-two-thirds padding-bottom-25">
+            <img src="images/logo-white.png">
+            </div>
+
+            <div>2307 Irving Street, Suite 228</div>
+            <div>San Francisco CA 94122</div>
+            <div>(800) 370-1416</div>
+            <div>info@canvasmedical.com</div>
+        </div>
+        <div class="column-one-fourth"></div>
+        <div class="column-one-fourth line-height-two">
+          <div>COMPANY</div>
+          <div>About Us</div>
+          <div>Blog</div>
```

Comment:
Do not use `div` elements as text elements. These are meant to be links, they should be `a` elements.